docs/cli/example_config.toml

```diff
@@ -8,7 +8,7 @@ verbosity = 3                   # Logging verbosity for the server (5=trace|4=de
 vmdebug = false                 # Record information useful for VM and contract debugging
 datadir = "var/lib/bor"         # Path of the data directory to store information
 ancient = ""                    # Data directory for ancient chain segments (default = inside chaindata)
-"db.engine" = "leveldb"           # Used to select leveldb or pebble as database (default = leveldb)
+"db.engine" = "pebble"           # Used to select leveldb or pebble as database (default = leveldb)
```

Comment:
can we also change  the comment?

Reply:
`state.scheme` is missing here. Maybe worth adding it here for consistency. Also, not here, but somewhere we can mention that `PBSS` it is not available for archive nodes?

Reply:
Or maybe fail starting archive node in PBSS with an error?

Reply:
Thanks for the suggestions! Added `state.scheme` here. Also add a check that will fail when PBSS and archive is enabled at the same time.